docs/building-spark.md

```diff
@@ -163,11 +164,9 @@ the `spark-parent` module).
 
 Thus, the full flow for running continuous-compilation of the `core` submodule may look more like:
 
-```
- $ mvn install
- $ cd core
- $ mvn scala:cc
-```
+    $ mvn install
```

Comment:
It's merged already, but for the sake of completeness I'm answering now - yes, it's *properly* code-formatted.